package.json

```diff
@@ -10,10 +10,10 @@
   },
   "author": "PDFTron Systems Inc.",
   "devDependencies": {
-    "@pdftron/webviewer-downloader": "^1.0.7",
+    "@pdftron/webviewer": "^8.0.0",
```

Comment:
The `download-webviewer` script needs to be removed and `postinstall` script will need to be changed as well:

https://github.com/PDFTron/webviewer-annotations-nodejs-sample/blob/master/package.json
https://github.com/PDFTron/webviewer-annotations-nodejs-sample/blob/master/tools/copy-webviewer-files.js